_data/chaithanya21/chaithanya21.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Chaithanya Kumar A
+institution: M.S Ramaiah University Of Applied Sciences
+profile_pic: chaithanya.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `chaithanya.jpg` to `chaithanya21.jpg` in the markdown file.
2. Rename `chaithanya21.jpg` to `chaithanya.jpg`.
I'd suggest you change the name in the markdown file!